### PR TITLE
use http to post to discord

### PR DESCRIPTION
**Pull request by ryjm**
The gateway method is slow af for some reason.

also bolded discord names and added support for embeds/images (sorta)

### CODE REVIEW

File: faux/faux.py [hunk 1]
```diff
@@ -59,13 +62,41 @@ async def on_message(self, message):
...
                     channel = matching_channels[0]
 
-                    self.urbit_client.post_message(
-                        self.group["urbit_ship"],
-                        channel["urbit_channel"],
-                        {"text": "%s: %s" % (message.author.display_name, message.content)}
-                    )
-
-class FauxDiscordPoster(discord.Client):
+                    if (len(message.attachments) > 0):
```
Comment (midsum-salrux) [inline faux/faux.py]: Unnecessary parens here

File: faux/faux.py [hunk 2]
```diff
@@ -59,13 +62,41 @@ async def on_message(self, message):
...
+                        messageAttachment = message.attachments[0].url
```
Comment (midsum-salrux) [inline faux/faux.py]: Use snake_case per PEP 8

File: faux/faux.py [hunk 3]
```diff
@@ -59,13 +62,41 @@ async def on_message(self, message):
...
+                        print(messageAttachment)
```
Comment (midsum-salrux) [inline faux/faux.py]: Looks like a debugging print was left in

File: faux/faux.py [hunk 4]
```diff
@@ -59,13 +62,41 @@ async def on_message(self, message):
...
+                        if (messageAttachment):
```
Comment (midsum-salrux) [inline faux/faux.py]: Unnecessary parens here

**Review by midsum-salrux — APPROVED** [review]
LGTM aside from some tiny style issues.

**Comment by ryjm** [comment]
Added some more stuff, the gif thing is a bit hacky but it does the job.

File: faux/faux.py [hunk 5]
```diff
@@ -137,7 +161,8 @@ async def urbit_action(message, _):
 
                     self.message = {"channel_id": channel["discord_channel_id"],
                                     "content": "%s" % (message.full_text)}
-                    await self.send_message(
+                    if message.author == "littel-wolfur":
```
Comment (midsum-salrux) [inline faux/faux.py]: :eyes:

File: faux/faux.py [hunk 6]
```diff
@@ -61,41 +62,64 @@ async def on_message(self, message):
...
-                        print(messageAttachment)
-                        if (messageAttachment):
+                        url = message.attachments[0].url
+                        result = {"text": f'__{author}__: {parsed}'}
+                        self.urbit_client.post_message(
+                            self.group['urbit_ship'],
+                            channel["urbit_channel"],
+                            result
+                            )
+                        if (url):
```
Comment (midsum-salrux) [inline faux/faux.py]: Parens

File: faux/faux.py [hunk 7]
```diff
@@ -61,41 +62,64 @@ async def on_message(self, message):
...
                             self.urbit_client.post_message(
                                 self.group['urbit_ship'],
                                 channel["urbit_channel"],
-                                {"text": "__%s:__ %s" % (message.author.display_name, message.content),
-                                 "url": messageAttachment}
+                                {
+                                 "url": url
+                                 }
```
Comment (midsum-salrux) [inline faux/faux.py]: Alignment

**Review by midsum-salrux — APPROVED** [review]

**Pull request merged**